ice40/utils/ice40_import_routing_from_icebox.py

```diff
@@ -437,6 +437,36 @@ def add_ram_pin(rw, sig, ind=None):
             add_ram_pin('W', 'DATA', ind)
             add_ram_pin('', 'MASK', ind)
 
+    # BLK_TL-RAM
```

Comment:
Can we log an issue about reducing duplicate between this stuff and the `ram_pin_offset` below?

Reply:
#179